### ISSUE TITLE
EGS installedto D:\Program Files\Epic Games\AlienIsolation\Alienisolation'   Cathode editor to fails to launch

### ISSUE THREAD

**Issue by ghramsey** [MattFiler/OpenCAGE#138]
After having some windows issues I removed most of my programs, repaired windows etc.  
When putting it all back together I reinstalled EGS  & Alien Isolation,  
For some reason Epic launcher decided the folder was "full".    
It had a myriad of folders for various mods I've been playing with.
,
Unbeknownst to me it installed to D:\Program Files\Epic Games\AlienIsolation\alienisolation  
The EGS launcher did not indicate this subfolder would be made BTW.    It showes in the path the "correct" default directory path.

OpenCage front end launches fine, but trying to open the cathode editor on this non standard
 install nothing happened. It just failed to launch.   The configuration editor however opened fine.
No error message about the failure. I figured the opencage install was borked and tried to reinstall it and got a file in use error.   
Cathode editor was running and had to be killed via task manager.   
Reinstalling open cage did not fix the  issue it was then I realized  EGS made this subfolder.

This is probably something nobody else will ever run into, but since it didn't even throw an error message I'm reporting it.
IDK if it's a bug but it should at least throw an error or exit quietly.

**Comment by MattFiler** [comment]
Interesting - usually the editor will fail silently if launched from the wrong directory or with the wrong parameters. I'd recommend:

1) Delete OpenCAGE exe & json file in the game's root folder
2) Delete the /DATA/MODS/ and /DATA/MODTOOLS/ folders
3) Re-download OpenCAGE
4) Make sure the game's root folder has write privelages
4) Copy OpenCAGE to the root folder & run again

If you still have issues let me know. I don't store any info in AppData etc so doing that should be a complete reset.

**Comment by ghramsey** [comment]
Thank you.

I went and fought w/ EGS to put the game in the right folder.   Took 3 tries to get it to stop adding an extra subfolder.  
It was even showing a folder name with some sort of GUID string, but still named it  \alienisolation.
I finally removed the entire epic games sub directoy tree and let it create it.  That fixed it.  
EGS is as so many have noted very buggy.

Hopefully, I'll never encounter this problem again. 

If I do I'll take note of how to correct it.
